sentry-samples/sentry-samples-spring-boot-jakarta/src/main/java/io/sentry/samples/spring/boot/jakarta/PersonService.java

```diff
@@ -18,12 +20,15 @@ public class PersonService {
   private static final Logger LOGGER = LoggerFactory.getLogger(PersonService.class);
 
   private final JdbcTemplate jdbcTemplate;
+  private final CacheManager cacheManager;
   private int createCount = 0;
 
-  public PersonService(JdbcTemplate jdbcTemplate) {
+  public PersonService(JdbcTemplate jdbcTemplate, CacheManager cacheManager) {
     this.jdbcTemplate = jdbcTemplate;
+    this.cacheManager = cacheManager;
   }
 
+  @Cacheable(value = "people", key = "{ #root.methodName, #person.firstName, #person.lastName }")
```

Comment:
If no key is defined, `@Cacheable` will use the object itself as key, so beware of sending PII to Sentry.